examples/config.yaml

```diff
@@ -46,6 +46,7 @@ odfi:
   storage:
     cleanupLocalDirectory: true
     keepRemoteFiles: false
+    removeZeroByteFilesAfter: 10
```

Comment:
We need to add this into `./docs/config.md` as well.

Reply:
The way the code reads we need `keepRemoteFiles: false` and `removeZeroByteFilesAfter > 0` in order for these files to be removed. Yes?

I'm torn if we should require `keepRemoteFiles: false` with this. The original problem is our ODFI creates zero-bytes files, but we want to keep the non zero-byte files and just delete the zero-byte files.

Reply:
> We need to add this into `./docs/config.md` as well.

Can do.

Reply:
> The way the code reads we need `keepRemoteFiles: false` and `removeZeroByteFilesAfter > 0` in order for these files to be removed. Yes?
> 
> I'm torn if we should require `keepRemoteFiles: false` with this. The original problem is our ODFI creates zero-bytes files, but we want to keep the non zero-byte files and just delete the zero-byte files.

That is correct. 

I think you are right. We may want to run a `deleteEmptyFiles` regardless of `keepRemoteFiles`

Reply:
This example needs to be "10m" for paygate to read it properly.

Reply:
agreed.